lardataobj/RecoBase/OpFlash.h

```diff
@@ -10,36 +10,55 @@
 #define OPFLASH_H
 
 #include <vector>
+#include <limits> // std::numeric_limits<>
+
 
 namespace recob {
 
   // subevents are groupings of OpDet pulse peaks. Each peak comes from an OpDet and
   // has a certain number of PE; each subevent has a time associated with it
   class OpFlash {
     public:
-      OpFlash(); // Default constructor
+      
+      /// Special value used for absence of center location information.
+      static constexpr double NoCenter = std::numeric_limits<double>::max();
+      
+      OpFlash() = default;
 
 private:
 
-      double                fTime;         // Time relative to trigger
-      double                fTimeWidth;    // Width of the flash in time
-      double                fAbsTime;      // Time by PMT readout clock
-      unsigned int          fFrame;        // Frame number
-      std::vector< double > fPEperOpDet;   // Number of PE on each PMT
-      std::vector< double > fWireCenters;  // Geometric center in each view
-      std::vector< double > fWireWidths;   // Geometric width in each view
-      double                fYCenter;      // Geometric center in y
-      double                fYWidth;       // Geometric width in y
-      double                fZCenter;      // Geometric center in z
-      double                fZWidth;       // Geometric width in z
-      double                fFastToTotal;  // Fast to total light ratio
-      bool                  fInBeamFrame;  // Is this in the beam frame?
-      int                   fOnBeamTime;   // Is this in time with beam?
+      double                fTime { 0.0 }; ///< Time on @ref DetectorClocksHardwareTrigger "trigger time scale" [us]
```

Comment:
Do you understand why this data member is default-initialized whereas most of the other non-class types are not?  I suggest either initializing all of the non-class types, or none of them.  (Granted, this PR does not intend to address the initialization issue.)